<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerModelsModuleWidget</class>
 <widget class="qSlicerWidget" name="qSlicerModelsModuleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>412</width>
    <height>796</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Models</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="ctkExpandableWidget" name="ResizableFrame">
     <property name="orientations">
      <set>Qt::Vertical</set>
     </property>
     <property name="sizeGripInside">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="allModelsHorizontalLayout">
        <item>
         <widget class="ctkSearchBox" name="FilterModelSearchBox">
          <property name="placeholderText">
           <string>Filter by name...</string>
          </property>
          <property name="showSearchIcon">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="hideAllModelsButton">
          <property name="toolTip">
           <string>Turn the visibility off on all models (does not include hierarchies)</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>:/Icons/Small/SlicerInvisible.png</normaloff>:/Icons/Small/SlicerInvisible.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="showAllModelsButton">
          <property name="toolTip">
           <string>Turns visibility on for all models (does not include hierarchies)</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>:/Icons/Small/SlicerVisible.png</normaloff>:/Icons/Small/SlicerVisible.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="qMRMLSubjectHierarchyTreeView" name="SubjectHierarchyTreeView">
        <property name="dragDropMode">
         <enum>QAbstractItemView::InternalMove</enum>
        </property>
        <property name="indentation">
         <number>12</number>
        </property>
        <property name="editMenuActionVisible">
         <bool>false</bool>
        </property>
        <property name="multiSelection">
         <bool>true</bool>
        </property>
        <property name="hideChildNodeTypes">
         <stringlist notr="true">
          <string>vtkMRMLAnnotationNode</string>
         </stringlist>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="InformationButton">
     <property name="text">
      <string>Information</string>
     </property>
     <property name="collapsed">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="qMRMLModelInfoWidget" name="MRMLModelInfoWidget"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="DisplayButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Display</string>
     </property>
     <property name="collapsed">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="qMRMLModelDisplayNodeWidget" name="ModelDisplayWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="clippingConfigurationButtonVisible">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ctkCollapsibleGroupBox" name="ColorLegendCollapsibleGroupBox">
        <property name="toolTip">
         <string>Color legend for the Color Table selected in Scalars section.</string>
        </property>
        <property name="title">
         <string>Color Legend</string>
        </property>
        <property name="collapsed">
         <bool>true</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="qMRMLColorLegendDisplayNodeWidget" name="ColorLegendDisplayNodeWidget"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="qMRMLNodeComboBox" name="ClipModelsNodeComboBox">
     <property name="nodeTypes">
      <stringlist notr="true">
       <string>vtkMRMLClipModelsNode</string>
      </stringlist>
     </property>
     <property name="showHidden">
      <bool>true</bool>
     </property>
     <property name="addEnabled">
      <bool>false</bool>
     </property>
     <property name="removeEnabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="ClippingButton">
     <property name="text">
      <string>Clipping Planes</string>
     </property>
     <property name="collapsed">
      <bool>true</bool>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="ClipSelectedModelLabel">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Clip selected model:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="ClipSelectedModelCheckBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="qMRMLClipNodeWidget" name="MRMLClipNodeWidget"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLClipNodeWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLClipNodeWidget.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLModelInfoWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLModelInfoWidget.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLColorLegendDisplayNodeWidget</class>
   <extends>qMRMLWidget</extends>
   <header>qMRMLColorLegendDisplayNodeWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLModelDisplayNodeWidget</class>
   <extends>qMRMLWidget</extends>
   <header>qMRMLModelDisplayNodeWidget.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLSubjectHierarchyTreeView</class>
   <extends>QTreeView</extends>
   <header>qMRMLSubjectHierarchyTreeView.h</header>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>ctkCollapsibleGroupBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkExpandableWidget</class>
   <extends>QFrame</extends>
   <header>ctkExpandableWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkSearchBox</class>
   <extends>QLineEdit</extends>
   <header>ctkSearchBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>qSlicerModelsModuleWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>ClipModelsNodeComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>492</x>
     <y>588</y>
    </hint>
    <hint type="destinationlabel">
     <x>465</x>
     <y>740</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerModelsModuleWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>ModelDisplayWidget</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ClipModelsNodeComboBox</sender>
   <signal>currentNodeChanged(vtkMRMLNode*)</signal>
   <receiver>MRMLClipNodeWidget</receiver>
   <slot>setMRMLClipNode(vtkMRMLNode*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>255</x>
     <y>740</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>780</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerModelsModuleWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>SubjectHierarchyTreeView</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>267</x>
     <y>157</y>
    </hint>
    <hint type="destinationlabel">
     <x>288</x>
     <y>56</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showAllModelsButton</sender>
   <signal>clicked()</signal>
   <receiver>qSlicerModelsModuleWidget</receiver>
   <slot>showAllModels()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>613</x>
     <y>13</y>
    </hint>
    <hint type="destinationlabel">
     <x>528</x>
     <y>-9</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hideAllModelsButton</sender>
   <signal>clicked()</signal>
   <receiver>qSlicerModelsModuleWidget</receiver>
   <slot>hideAllModels()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>579</x>
     <y>11</y>
    </hint>
    <hint type="destinationlabel">
     <x>207</x>
     <y>-9</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerModelsModuleWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>ColorLegendDisplayNodeWidget</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>159</x>
     <y>394</y>
    </hint>
    <hint type="destinationlabel">
     <x>194</x>
     <y>1032</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>hideAllModels()</slot>
  <slot>showAllModels()</slot>
 </slots>
</ui>
